beetsplug/deezer.py

```diff
@@ -129,14 +160,18 @@ def _get_track(self, track_data):
         return TrackInfo(
             title=track_data['title'],
             track_id=track_data['id'],
+            deezer_track_id=track_data['id'],
+            isrc=track_data.get('isrc'),
             artist=artist,
             artist_id=artist_id,
             length=track_data['duration'],
             index=track_data.get('track_position'),
             medium=track_data.get('disk_number'),
+            deezer_track_rank=track_data.get('rank'),
             medium_index=track_data.get('track_position'),
             data_source=self.data_source,
             data_url=track_data['link'],
+            deezer_updated=time.time(),
```

Comment:
I don't 100% see yet the need for this field… we only seem to be writing to it, never reading from it. Is it useful for some future functionality?

Reply:
Yes, it is useful to only update based on time elapsed. For example, update the tracks that have not been updated in the last 2 months. In the absence of this information, we will have to update all the tracks.

Reply:
Got it; that makes sense!